src/format.rs

```diff
@@ -41,36 +41,63 @@ impl fmt::Display for FormattedDuration {
     }
 }
 
+// `HumanDuration` should be as intuitively understandable as possible.
+// So we want to round, not truncate (otherwise 1 hour and 59 minutes
+// would display an ETA of "1 hour" which underestimates the time remaining
+// by a factor 2.
+//
+// To make the precision more uniform, we avoid displaying "1 unit"
+// (except for seconds), because it would be displayed for an important
+// time relatively to the unit itself.  Instead, when we arrive around
```

Comment:
I find "for an important time" a bit hard to understand. Can you phrase that more simply? Also, please use single spaces after periods.